cmsmon-tools.spec

```diff
@@ -1,4 +1,4 @@
-### RPM cms cmsmon-tools 0.5.35
+### RPM cms cmsmon-tools go-0.6.3
```

Comment:
I would suggest putting `go-` part into the URL (line 17), and keep the version purely numeric, but will let @smuzaffar have the final say.